common/main/java/com/couchbase/lite/FullTextIndex.java

```diff
@@ -67,7 +67,7 @@ public FullTextIndex ignoreAccents(boolean ignoreAccents) {
     String getLanguage() { return language; }
 
     @Override
-    boolean isIgnoringDiacritics() { return ignoreDiacritics; }
+    boolean isIgnoringAccents() { return ignoreDiacritics; }
```

Comment:
@pasin : should this be public?   Is it part of the API?

Reply:
I crossed check with iOS and we haven't made them public.

Reply:
@bmeike @pasin I'm seeing this API as public in [ObjC](https://github.com/couchbase/couchbase-lite-ios/blob/master/Objective-C/CBLFullTextIndex.h#L30) and [Swift](https://github.com/couchbase/couchbase-lite-ios/blob/master/Swift/Index.swift#L81).

Reply:
The API spec doesn't say that it's public but we could make it public as Objective-C is already public. I will make explicitly change to the API spec and inform all the parties.